nodes/celestia-app.md

```diff
@@ -13,8 +13,45 @@ import mochaVersions from '/.vitepress/constants/mocha_versions.js'
 import coffeeVersions from '/.vitepress/constants/coffee_versions.js'
 </script>
 
-This tutorial will guide you through installing celestia-app. This
-tutorial presumes you completed the steps in
+This tutorial will guide you through installing celestia-app, both with
+[a pre-built binary](#installing-a-pre-built-binaries) and
+[from source](#building-binary-from-source).
+
+## Installing a pre-built binaries
```

Comment:
[optional] singular binary b/c a user will install a single binary not multiple 🤷 

```suggestion
## Installing a pre-built binary
```

Reply:
oops 👀